config/zowe-api-dev/template-stc.jcl

```diff
@@ -0,0 +1,13 @@
+//RUNJAVA  PROC SRVRPATH='{{{user.zosTargetDir}}}'
```

Comment:
I guess STC is not supported by an open-sourced version of the API Dev tool...

Reply:
maybe not, but it can be ready :)